transit_odp/timetables/dataframes.py

```diff
@@ -179,6 +196,146 @@ def journey_pattern_sections_to_dataframe(sections):
     return timing_links
 
 
+def vehicle_journeys_to_dataframe(
+    standard_vehicle_journeys, flexible_vechicle_journeys
+):
+    all_vechicle_journeys = []
+    if standard_vehicle_journeys is not None:
+        for vehicle_journey in standard_vehicle_journeys:
+            departure_time = vehicle_journey.get_element(["DepartureTime"]).text
+            journey_pattern_ref_element = vehicle_journey.get_element_or_none(
+                ["JourneyPatternRef"]
+            )
+            journey_pattern_ref = ""
+            if journey_pattern_ref_element:
+                journey_pattern_ref = journey_pattern_ref_element.text
+            line_ref = vehicle_journey.get_element(["LineRef"]).text
+            journey_code_element = vehicle_journey.get_element_or_none(
+                ["Operational", "TicketMachine", "JourneyCode"]
+            )
+            journey_code = ""
+            if journey_code_element:
+                journey_code = journey_code_element.text
+
+            vehicle_journey_code = vehicle_journey.get_element(
+                ["VehicleJourneyCode"]
+            ).text
+            service_ref = vehicle_journey.get_element(["ServiceRef"]).text
+
+            all_vechicle_journeys.append(
+                {
+                    "departure_time": departure_time,
+                    "journey_pattern_ref": "-".join([service_ref, journey_pattern_ref]),
+                    "line_ref": line_ref,
+                    "journey_code": journey_code,
+                    "vehicle_journey_code": vehicle_journey_code,
+                }
+            )
+
+    if flexible_vechicle_journeys is not None:
+        for vehicle_journey in flexible_vechicle_journeys:
+            line_ref = vehicle_journey.get_element(["LineRef"]).text
+            journey_pattern_ref = vehicle_journey.get_element(
+                ["JourneyPatternRef"]
+            ).text
+            vehicle_journey_code = vehicle_journey.get_element(
+                ["VehicleJourneyCode"]
+            ).text
+            service_ref = vehicle_journey.get_element(["ServiceRef"]).text
+
+            all_vechicle_journeys.append(
+                {
+                    "departure_time": None,
+                    "journey_pattern_ref": "-".join([service_ref, journey_pattern_ref]),
+                    "line_ref": line_ref,
+                    "journey_code": None,
+                    "vehicle_journey_code": vehicle_journey_code,
+                }
+            )
+
+    return pd.DataFrame(all_vechicle_journeys)
+
+
+def operating_profile_to_df(operating_profiles, op_type):
+    serviced_orgs_refs_df = []
+    if operating_profiles is not None and op_type == "VehicleJourney":
+        for operating_profile in operating_profiles:
+            serviced_organisation_day_type = operating_profile.get_element_or_none(
```

Comment:
The values read in this line and the other elements seem to match for `VehicleJourney` and `Services`. We could remove the if block for `Services` and the `and` condition from the `if` condition

Reply:
Done